### PR TITLE
Add 404 support

### PR DESCRIPTION
**Pull request by quaxsze**
fix https://github.com/etalab/csvapi/issues/68

### CODE REVIEW

**Comment by quaxsze** [comment]
Is checking for 404 enough or should we check anything different from 2xx status code?

File: csvapi/parseview.py [hunk 1]
```diff
@@ -23,6 +23,8 @@ async def do_parse(url, urlhash, encoding, storage, logger, sniff_limit, max_fil
         try:
             async with aiohttp.ClientSession() as session:
                 async with session.get(url) as resp:
+                    if resp.status == 404:
```
Comment (abulte) [inline csvapi/parseview.py:26]: I would use `raise_for_status` here and catch + reraise APIError

**Review by abulte — COMMENTED** [review]
simple enough 👌 add a test?

**Review by abulte — APPROVED** [review]
Nice! You can release and deploy 💪

**Pull request merged**